content/blog/2021-11-17-exp-versioning.md

```diff
@@ -0,0 +1,243 @@
+---
+title: Don't Just Track Your Experiments, Version Them
+date: 2021-11-17
+description: >
+  Versioning ML experiments in DVC helps you log, organize, and reproduce
+  experiments easier than ever.
+descriptionLong: >
+  With DVC experiments, you can take ML experiment tracking to the next level,
+  making it easy and powerful to save, compare, and reproduce experiments at
+  scale.
+picture:
+pictureComment:
+author: dave_berenbaum
+commentsUrl:
+tags:
+  - MLOps
+  - DVC
+  - Experiments
+  - Experiment Tracking
+  - Experiment Versioning
+---
+
+Traditional software version control is not enough for tracking machine learning
```

Comment:
Good and strong statement. I wish we have something similar in the description and probably in the title.